learning_journal_basic1/views.py

```diff
@@ -4,10 +4,28 @@
 HERE = os.path.dirname(__file__)
 
 
-def home_page(request):
-    imported_text = open(os.path.join(HERE, 'sample.txt')).read()
+def list_view(request):
+    imported_text = open(os.path.join(HERE, './templates/index.html')).read()
```

Comment:
You don't need to use `./` in front here. `os.path.join(HERE, ...)` already gives you that file path to the present directory.